parser/src/main/scala/jawn/NullFacade.scala

```diff
@@ -25,7 +25,6 @@ object NullFacade extends Facade[Unit] {
   def jnull(): Unit = ()
   def jfalse(): Unit = ()
   def jtrue(): Unit = ()
-  def jnum(s: String): Unit = ()
-  def jint(s: String): Unit = ()
+  def jnum(s: String, decIndex: Int, expIndex: Int): Unit = ()
```

Comment:
Can we add a comment here as to the meaning of these strings? Also what about keeping jnum and jint with 1 param which are final and call jnum with 3 params appropriately.